commands/inventory/list_agents.go

```diff
@@ -0,0 +1,164 @@
+// pmm-admin
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package inventory
+
+import (
+	"strings"
+
+	"github.com/AlekSi/pointer"
+	"github.com/percona/pmm/api/inventorypb/json/client"
+	"github.com/percona/pmm/api/inventorypb/json/client/agents"
+
+	"github.com/percona/pmm-admin/commands"
+)
+
+var listAgentsGenericResultT = commands.ParseTemplate(`
```

Comment:
Is that `Generic` part copied from Generic Node template? That's Node type, not just word.

Reply:
yeah, my fault